src/System.Security.Cryptography.Algorithms/tests/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-24117-00",
     "System.IO": "4.1.0-rc3-24117-00",
+    "System.IO.FileSystem": "4.0.1-rc3-24117-00",
```

Comment:
PlatformDetection.cs was added to detect the version of Windows, and that uses the File class.